scripts/db-exporter/build.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# The script must be invoked with the concordium-node source directory as working directory.
+
+# Expected parameters:
+# - VERSION
+
+version="${VERSION}"
+
+rust_toolchain_version=1.62
+protoc_version=3.15.3
+
+# Install dependencies.
+
+apt-get update && \
+	DEBIAN_FRONTEND=noninteractive apt-get -y install \
+		curl \
+		libunbound-dev \
+		libprotobuf-dev \
+		libssl-dev \
+		pkg-config \
+		libnuma-dev \
+		libgmp-dev \
+		liblmdb0 \
+		postgresql-server-dev-all \
+		locales \
+        unzip \
+		liblmdb-dev
+
+# Install protobuf
+
+curl -L https://github.com/protocolbuffers/protobuf/releases/download/v${protoc_version}/protoc-${protoc_version}-linux-x86_64.zip -o protoc.zip
+unzip protoc.zip bin/protoc -d /usr/
+rm protoc.zip
+
+# Install Haskell.
+
+curl -sSL https://get.haskellstack.org/ | sh
+
+# Install Rust.
+
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+source "$HOME/.cargo/env"
+rustup set profile minimal
+rustup default "$rust_toolchain_version"
+
+# - 'database-exporter'
+
+stack build \
+	--stack-yaml=./concordium-consensus/stack.yaml \
+	--flag=concordium-consensus:-dynamic \
+	--copy-bins \
+	--local-bin-path=. \
+	concordium-consensus:database-exporter
+
+# Build Debian packages.
+
+build_dir=/build
+database_exporter_build_dir="$build_dir/database-exporter_$version"
+
+# - 'database-exporter'
+
+(
+
+mkdir -p "$database_exporter_build_dir"
+cd "$database_exporter_build_dir"
+mkdir -p etc/default
+mkdir -p usr/lib/systemd/system
+mkdir -p usr/bin
+mkdir -p DEBIAN
+
+cat <<EOF > etc/default/database-exporter
+BUCKET_NAME=
+CF_DISTRIBUTION_ID=
+CHUNK_SIZE=
+EOF
+
+cat <<EOF > DEBIAN/control
+Package: database-exporter
+Version: $version
+Section: extra
+Priority: optional
+Architecture: amd64
+Depends: debconf ( >= 1.5.73 ), debhelper ( >= 10 ), libunbound2 ( >= 1.6.7 ) | libunbound8 ( >= 1.9.4 ), openssl ( >= 1.1.1 ), libgmp10 ( >= 6.1.2 ), libpq-dev ( >= 9.5.19)
```

Comment:
I think unbound should not be necessary, and hasn't been for some time now. Neither should openssl.

Reply:
Thank you, they have been removed